modules/KalturaSupport/resources/mw.KWidgetSupport.js

```diff
@@ -302,10 +301,12 @@ mw.KWidgetSupport.prototype = {
 
 					if (action.pattern && action.replacement) {
 						var regExp=new RegExp(action.pattern, "i");
-						if (!this.originalServiceUrl){
-							this.originalServiceUrl = mw.config.get('Kaltura.playManifestServiceUrl');
+						var originalServiceUrl = window.parent.mw.getConfig('Kaltura.originalServiceUrl');
```

Comment:
more elegant-
`var originalServiceUrl = window.parent.mw.getConfig('Kaltura.originalServiceUrl') || mw.config.get('Kaltura.playManifestServiceUrl');`

Reply:
it not the same. the propose is to save the originalServiceUrl because the playManifestServiceUrl is override. the bug was that it used to take the playManifestServiceUrl after it was set and it cause to use the wrong url and so the player didnt play